spikingjelly/activation_based/cuda_kernel/neuron_kernel/single_step/base.py

```
@@ -1,13 +1,13 @@
-import logging
+from spikingjelly.logger import logger
 import math
 
 import torch
 
 try:
     import cupy
 except BaseException as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)